src/styles/utils/_variables.scss

```
@@ -0,0 +1,6 @@
+$days-in-week: 7;
+$day-size: 100px;
+$days-gap: 1px;
+$calendar-horizontal-padding: 10px;
```

I see you are using this var as well for setting vertical padding so I would call this: 
```suggestion
$calendar-padding: 10px;
```

---

I renamed variable.